ocaml-lsp-server/src/subarray.mli

```diff
@@ -0,0 +1,14 @@
+type 'a t
+
+val make : 'a array -> start:int -> ?length:int -> unit -> 'a t
```

Comment:
Could you could it `pos` and `len` as customary. Also, can we document the behavior when `len` is absent?